### ISSUE TITLE
Currently Active Tasks component epic

### ISSUE THREAD

**Issue by lowellrex** [department-of-veterans-affairs/caseflow#8142]
_Lowell: The final version of case timeline (mocks in [this mural](https://app.mural.co/t/workqueue2001/m/workqueue2001/1531849576358/38b753bff1e25963a76d8ebecf9679d6445c39b3)), we will include the incomplete tasks above the completed tasks in a continuous linear view. In order to get there we need to be able to view multiple tasks at once. I think the mocks provide a good example of how this could look. I propose we move the task information and actions dropdown out of the case snapshot area with this ticket and move it into this lockup._

_An open question is what case snapshot looks like without the task information. Perhaps docket type and AOD/CAVC (and VLJ and attorney information) get moved into [the proposed universal case title](https://github.com/department-of-veterans-affairs/caseflow/issues/6963)? @sneha-pai what do you think of this approach?_ [Resolved by Universal Case title]

## Acceptance criteria
- All currently active tasks are included in a lockup below the universal case title

- A case will be defined as  "currently active" if it is assigned to an individual or team(except colocated team) and is not on hold due to a blocking task.

- If a case is on hold for any user/team (due to a blocking task), it will appear in Case Timeline, with a grey dot (incomplete) rather than a green checkmark dot.

- Cases on-hold for the colocated team will not appear in the case timeline.

- Once a case is marked complete, it is no longer currently active, and it will now appear as a green-check-marked case in Case Timeline.

- If a colocated individual places a case on hold (with a window of time), and that task has no child task, that case will continue to appear as Currently Active.

#### Scenario: No Task assigned to current user (viewer just sees task list, but no "actions" dropdown.
![current-task-viewer-not-assigned](https://user-images.githubusercontent.com/31224257/49607317-96b01200-f963-11e8-8b98-63e359af6511.png)

#### Scenario: One task assigned to current user (colocated individual)
![current-task-single-task-assigned-to-colocated-user](https://user-images.githubusercontent.com/31224257/49608674-3fac3c00-f967-11e8-8614-567fadf1db55.png)

#### Scenario: When colocated is done with their, the attorney's task is now unblocked (viewing as attorney)
![current-task-single-task-assigned-to-current-user-attorney-unblocked](https://user-images.githubusercontent.com/31224257/49607326-a0397a00-f963-11e8-9db4-18ef6a2ee33f.png)

#### If colocated were to assign this task to another team, the colocated task would be on hold and blocked (viewing as AOD individual)
![current-task-single-task-assigned-to-aod-individual](https://user-images.githubusercontent.com/31224257/49607330-a3cd0100-f963-11e8-85b0-8cf1f5f0abd4.png)

#### Scenario: Two tasks assigned to current user (default view)
![current-task-multiple-tasks-assigned-to-current-user-task-instructions-closed](https://user-images.githubusercontent.com/31224257/49607337-a7f91e80-f963-11e8-8588-663d4083072d.png)

#### Scenario: Two tasks assigned to current user + Task instructions are open
![current-task-multiple-tasks-assigned-to-current-user-task-instructions-open](https://user-images.githubusercontent.com/31224257/49607349-afb8c300-f963-11e8-96ad-a1d26213cccf.png)

Please see https://app.mural.co/t/workqueue2001/m/workqueue2001/1543851111020/bbd6df5b95ca9a9ad179cd991d40c9314f35c929 for more details about the active-task logic.

**Comment by lowellrex** [comment]
#8141 is the correct ticket for this epic.
